tests/data_import/test_model_aliases.py

```diff
@@ -0,0 +1,101 @@
+import pytest
+from django.core.exceptions import ValidationError
+
+from data_import.models import RespondingAgencyAlias
+from payroll.models import EmployerAlias
+
+
+@pytest.mark.django_db
+def test_responding_agency_preferred_alias(responding_agency):
+    agency = responding_agency.build()
+
+    old_alias = RespondingAgencyAlias.objects.create(responding_agency=agency, name='a_rose', preferred=True)
+
+    new_alias = RespondingAgencyAlias.objects.create(responding_agency=agency, name='by_any_other_name', preferred=True)
+
+    old_alias.refresh_from_db()
+
+    assert new_alias.preferred is True
+    assert old_alias.preferred is False
+
+
+@pytest.mark.django_db
+def test_responding_agency_unique_alias(responding_agency):
+    agency = responding_agency.build()
+
+    with pytest.raises(ValidationError):
+
+        RespondingAgencyAlias.objects.create(responding_agency=agency, name='a_rose')
```

Comment:
Can you move the first alias creation out of this `with pytest.raises` block, so it's more explicit which call you expect to fail? Ditto on the tests for unit and department aliases.